### DISCUSSION TITLE
BORIS v8.x for mac

### DISCUSSION THREAD

**Discussion by PavoPavan** [olivierfriard/BORIS#558] [Q&A]
Hi, is there a release plan for v8.x for mac?
It would be really helpful to me

Thanks

**Comment by olivierfriard** [comment]
Hi,
not yet for an issue with the MPV libraries on MacOS.
I am trying to fix this

Olivier

**Comment by Philippe-s-Levi** [comment]
did you try windows emulator ?

**Comment by olivierfriard** [comment]
Hi,
I just released a version of BORIS that should run natively on macOS with some limitations described on https://www.boris.unito.it/download_mac.
This is an experimental version so test it and let me know
